docs/access.rst

```diff
@@ -118,6 +118,12 @@ For example, if Kolibri is installed and started on a computer with the address
   * You can also use the ``ipconfig`` command on Windows or ``ifconfig`` command on Linux/OSX to find the externally visible IP address of the device running the Kolibri.
 
 
+.. warning::
+  * When limited to using older browsers it may be necessary to **explicitly include** the ``http://`` in front of the IP in the address bar, for example ``http://192.168.8.134:8080``. Many modern browsers may also take the IP address or the local network hostname as a search term and redirect you to Google. In other cases, browsers may try to use the secure **HTTPS** protocol by default and add an **S**  automatically (``httpS://``), which may also cause issues while running Kolibri.
```

Comment:
I don't thin `http://192.168.0.134:8080` is a good example. The browser will redirect to https when the port is not used. I.E: `http://192.168.0.134`will be redirect to `https://192.168.0.134` because the port is not added. If the port is added it keeps `http`.
That's why I mentioned the problem exists *only* when using port *80*, because this was the former implicit port and now it has been changed to *443* (https) by the browsers since they implemented https://en.wikipedia.org/wiki/HTTP_Strict_Transport_Security

Reply:
I'm not sure how browsers will implement this in the future, but I think the warning is a reasonable caution such that everyone can go ahead and make their own observations / troubleshooting with some pretty solid hints.

Reply:
We can address Jose's feedback by adapting the text slightly

```suggestion
  * When accessing Kolibri by IP, in many browsers it is necessary to **explicitly include** the ``http://`` in front of the IP in the address bar. For example, this might be ``http://192.168.8.134:8080`` for port ``8080`` or ``http://192.168.8.134`` for port ``80``. Omitting ``http://`` may result in undesirable redirects.
```

Reply:
I already made a suggestion for this, see below ;)